src/components/search-form/__tests__/SearchForm.test.js

```diff
@@ -124,8 +124,12 @@ describe('components/search-form/SearchForm', () => {
 
     test('should set the onClearHandler to the clear button onClick prop', () => {
         const wrapper = shallow(<SearchForm intl={intlShape} />).shallow();
-        const lodableComponent = wrapper.find('LoadableSearchActions').shallow();
-        const searchActions = lodableComponent.shallow();
+        // Sift through the nested HOCs to find the correct element
+        const searchActions = wrapper
+            .find('LoadableSearchActions')
+            .shallow()
+            .shallow()
+            .shallow();
```

Comment:
admittedly not my best work...I briefly looked into using `mount` and alternatives but I selfishly didn't want to spend too much time on this component and changes

Reply:
shouldn't we being use two `dive()`s here instead of two `shallow()`?

Reply:
Also should this test be here? shouldn't it be in a new test file for `SearchActions` this way you dont have to go through the HOC?

Reply:
> shouldn't we being use two dive()s here instead of two shallow()?

i mainly copied from the existing test since I wasn't sure the exact differences. but it looks like `dive()` is preferred?https://github.com/enzymejs/enzyme/issues/1798

i'll update this to `dives()`

Reply:
> Also should this test be here? shouldn't it be in a new test file for SearchActions this way you dont have to go through the HOC?

yeah this was one of the alternatives I was looking into but then moving it to a new `SearchActions` file I think sort of defeats the intention of the original test since it's testing the `onClearHandler` in `SearchForm` (the parent component)

Reply:
I think the ideal way to test it would be to convert this to RTL I guess and test the button click functionality? but then I was like: ahhh we'll save that for another day...